ledger.go

```diff
@@ -687,7 +690,11 @@ FINALIZE_ROUNDS:
 			Uint64("round_depth", finalized.End.Depth-finalized.Start.Depth).
 			Msg("Finalized consensus round, and initialized a new round.")
 
-		//go ExportGraphDOT(finalized, l.graph)
+			//go ExportGraphDOT(finalized, l.graph)
+
+		if l.FinalizeCh != nil {
```

Comment:
@iwasaki-kenta I added this as a hook to allow tests to wait for consensus round to finalize. It feels a bit like a dirty hack though. But so far it works better than just `time.Sleep`. Is there a better way to do this?

Reply:
Nvm, figured out a better way without having to alter `Ledger`. Refer to #150.